src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -557,6 +594,41 @@ class spell_warr_last_stand : public SpellScriptLoader
         }
 };
 
+// 12294 - Mortal Strike 7.1.5
+class spell_warr_mortal_strike : public SpellScriptLoader
+{
+public:
+    spell_warr_mortal_strike() : SpellScriptLoader("spell_warr_mortal_strike") { }
+
+    class spell_warr_mortal_strike_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_warr_mortal_strike_SpellScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(SPELL_WARRIOR_MORTAL_WOUNDS))
+                return false;
+            return true;
+        }
+
+        void HandleDummy(SpellEffIndex /*effIndex*/)
+        {
+            if (Unit* target = GetHitUnit())
```

Comment:
How can it hit objects if the target type is TARGET_UNIT_TARGET_ENEMY.. tell me Keader ^^